docs/superpowers/specs/2026-05-30-capgo-builder-onboarding-user-journey.html

```
@@ -0,0 +1,482 @@
+    </div>
+  </section>
+
+  <!-- WALKTHROUGH -->
+  <section id="walk">
+    <p class="kicker reveal">03 · The full walkthrough</p>
+    <h2 class="reveal">What it actually looks like, message by message</h2>
+    <p class="lead reveal">A realistic transcript for each platform — including the bumps. Dark bubbles are you; light bubbles are the AI; green notes are what's happening behind the scenes; coral notes are when you step away.</p>
+
+    <div class="tabs reveal">
+      <div class="tabbar" role="tablist">
+        <button role="tab" aria-selected="true" data-tab="ios">🍎 &nbsp;iOS — the full ride</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)